src/SonarScanner.MSBuild.Common/AnalysisConfig/AnalyzerSettings.cs

```diff
@@ -82,8 +82,8 @@ public AnalyzerSettings()
         /// </summary>
         /// <remarks>This includes analyzer assemblies and their dependencies</remarks>
         [XmlArray]
-        [XmlArrayItem("Path")]
-        public List<string> AnalyzerAssemblyPaths { get; set; }
+        [XmlArrayItem("AnalyzerPlugin")]
+        public List<AnalyzerPlugin> AnalyzerPlugins { get; set; }
```

Comment:
Changes the existing _AnalyzerSettings_ XML data class to serialize _AnalyzerPlugins_ instead of simple strings.